modules/api_reference/src/Plugin/OpenApiValidationException.php

```diff
@@ -2,12 +2,10 @@
 
 namespace Drupal\devportal_api_reference\Plugin;
 
-use Exception;
-
 /**
  * Exception for OpenApi validation errors.
  */
-class OpenApiValidationException extends Exception {
+class OpenApiValidationException extends \Exception {
```

Comment:
Hm, this should extend something module-specific, probably the RuntimeException because validation issues can only revel in runtime. (Or you can create a new exception class by implementing the module-specific exception interface.)